### ISSUE TITLE
A wrong of losing file.

### ISSUE THREAD

**Issue by machenxi6666** [xliangp/DIFFER#3]
Sorry to pardon you,I have to tell you that there is a wrong in the 6 lines of"data/__init__.py":"from data.datasets.prcc import PRCC",but your code hasn't gave the file.

**Comment by machenxi6666** [comment]
值得一提的是，在data/__init__.py第八行from data.datasets.ccvid import CCVID，但您提供的代码中没用这个文件

**Comment by xliangp** [comment]
We have fixed the missing file issue. Please let us know if there are any further issues.

**Comment by machenxi6666** [comment]
Thanks for your great contribution! I found an issue with the pre-trained weights for LTCC: they are 512-dimensional, but according to the config.yml, the checkpoint should use 768-dimensional features, and it's name is PRCC, this must be a wrong. Could you please upload the correct checkpints for LTCC? I would really appreciate it!

**Comment by amiragez0215** [comment]
> Thanks for your great contribution! I found an issue with the pre-trained weights for LTCC: they are 512-dimensional, but according to the paper, the model should use 768-dimensional features. Could you please upload the correct 768-dimensional weights? I would really appreciate it!

I apologize for the interruption. May I ask if you were able to successfully reproduce the accuracy results for LTCC? I was unable to reproduce them based on the contents of the `eva02_l_bio.yml` file.

**Comment by machenxi6666** [comment]
> > Thanks for your great contribution! I found an issue with the pre-trained weights for LTCC: they are 512-dimensional, but according to the paper, the model should use 768-dimensional features. Could you please upload the correct 768-dimensional weights? I would really appreciate it!
> 
> I apologize for the interruption. May I ask if you were able to successfully reproduce the accuracy results for LTCC? I was unable to reproduce them based on the contents of the `eva02_l_bio.yml` file.

I have reached 56.2% on LTCC. It still needs to develop more than 2% to reach author's result.

**Comment by amiragez0215** [comment]
> > > Thanks for your great contribution! I found an issue with the pre-trained weights for LTCC: they are 512-dimensional, but according to the paper, the model should use 768-dimensional features. Could you please upload the correct 768-dimensional weights? I would really appreciate it!
> > 
> > 
> > I apologize for the interruption. May I ask if you were able to successfully reproduce the accuracy results for LTCC? I was unable to reproduce them based on the contents of the `eva02_l_bio.yml` file.
> 
> I have reached 56.2% on LTCC. It still needs to develop more than 2% to reach author's result.

How did you manage that? I was only able to reach 50.3%. What are the hyperparameters for your LTCC?

**Comment by machenxi6666** [comment]
> > > > Thanks for your great contribution! I found an issue with the pre-trained weights for LTCC: they are 512-dimensional, but according to the paper, the model should use 768-dimensional features. Could you please upload the correct 768-dimensional weights? I would really appreciate it!
> > > 
> > > 
> > > I apologize for the interruption. May I ask if you were able to successfully reproduce the accuracy results for LTCC? I was unable to reproduce them based on the contents of the `eva02_l_bio.yml` file.
> > 
> > 
> > I have reached 56.2% on LTCC. It still needs to develop more than 2% to reach author's result.
> 
> How did you manage that? I was only able to reach 50.3%. What are the hyperparameters for your LTCC?

I found that the Rank-1 will develop more than 5% if we use 2 card rather than 1 card. So I use 2 × A800.
